interface/vsomeip/application.hpp

```diff
@@ -920,6 +927,8 @@ class application {
     virtual void remove_security_policy_configuration(uint32_t _uid,
                                                       uint32_t _gid,
                                                       security_update_handler_t _handler) = 0;
+                                                      
+    virtual boost::asio::io_service & get_io() = 0;
```

Comment:
Why?

Reply:
we need to add other file descriptors, instead of creating our own event loop. Therefore we have less resource usage and increase maintainability